renderer-macos/lib/stubs/BriskProgressIndicator.c

```diff
@@ -2,8 +2,17 @@
 
 NSProgressIndicator *ml_BriskProgressIndicator_make() {
   NSProgressIndicator *progressIndicator = [NSProgressIndicator new];
+  progressIndicator.style = NSProgressIndicatorStyleSpinning;
   [progressIndicator startAnimation:nil];
   progressIndicator.indeterminate = YES;
   retainView(progressIndicator);
   return progressIndicator;
 }
+
+double ml_BriskProgressIndicator_fittingWidth(NSProgressIndicator *indicator) {
+  return indicator.fittingSize.width;
```

Comment:
You're setting `ml_BriskProgressIndicator_fittingWidth_bc` in the stubs, but there's no impl for it. 🤷‍♂️

Reply:
I know! That's on purpose. If someone ever needs to use byte code versions they'll see the problem in the compile time.

Reply:
Neat. 👌